packages/ag-charts-community/src/module/optionsModule.ts

```diff
@@ -251,13 +252,17 @@ export class ChartOptions<T extends AgChartOptions = AgChartOptions> {
         this.processSeriesOptions(processedOptions, activeTheme);
         this.processMiniChartSeriesOptions(processedOptions, activeTheme);
 
-        this.annotationThemes = annotationsThemes;
-
         // Create isolated copy of options before we start mutations - this is performance sensitive,
         // so we aim to only do this once in the processing flow.
         processedOptions = deepClone(processedOptions, ChartOptions.OPTIONS_CLONE_OPTS);
 
-        this.processOptionsParams(processedOptions);
+        let publicParameters = activeTheme.getPublicParameters();
+        if (isPlainObject(processedOptions.theme) && processedOptions.theme.params) {
+            publicParameters = mergeDefaults(processedOptions.theme.params, publicParameters);
+        }
+
+        this.resolveOptionsParams(publicParameters, processedOptions);
+        this.resolveOptionsParams(publicParameters, this.annotationThemes);
```

Comment:
The primary change in this PR is here, moving the default theme params to the `getPublicParameters()` fn and applying them also the the annotation theme (which is split out of the processed options above).